wqflask/wqflask/templates/snp_browser.html

```diff
@@ -229,11 +225,23 @@ <h2>Variant Browser <a class="btn btn-primary" href="http://genenetwork.org/snpb
 
   <script type='text/javascript'>
       var json_rows = {{ table_rows|safe }};
+      var json_rows_count = {{ rows_count|safe }};
       var empty_columns = {{ empty_columns|safe }};
   </script>
 
   <script language="javascript">
 
+    var getParams = function(url) {
+      var parser = document.createElement('a');
+      parser.href = url;
+      var params = parser.search.substring(1);
+      console.log(params);
```

Comment:
**suggestion:** Let's avoid adding `console` statements upstream. I'd advise you remove them when pushing to upstream. `console` statements are great when troubleshooting, but it's the kind of thing we would not really want in a prod envoronment :)